cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -35,6 +35,8 @@ CLS_NAME(arrow)
 cls_handle_t h_class;
 cls_method_handle_t h_scan_op;
 
+/// \class RandomAccessObject
+/// \brief A RandomAccessFile implementation to read chunks of files.
```

Comment:
We can say, An interface to provide a file-like view over RADOS objects.

Reply:
Done.